e2e/_suites/fleet/ingest_manager_test.go

```diff
@@ -48,7 +48,6 @@ func setUpSuite() {
 	}
 	common.AgentVersionBase = v
 
-	common.TimeoutFactor = shell.GetEnvInteger("TIMEOUT_FACTOR", common.TimeoutFactor)
 	common.AgentVersion = shell.GetEnv("BEAT_VERSION", common.AgentVersionBase)
```

Comment:
We could do the same with any other variable defined in `common` that needs some initialization, such as this `common.AgentVersion`.